modules/client/package.json

```diff
@@ -40,7 +40,7 @@
     "semaphore": "^1.1.0",
     "typescript-fsa": "^3.0.0-beta-2",
     "typescript-fsa-reducers": "^1.1.0",
-    "web3": "^1.0.0-beta.36"
+    "web3": "=1.0.0-beta.36"
```

Comment:
Sometimes npm installs web3 version 1.0.0-beta.46 (especially after purging lock files) and this causes a bunch of typescript compilation errors. Being more strict w it's version prevents this.